openedx2zim/html_processor.py

```diff
@@ -149,6 +149,18 @@ def download_images_from_html(self, html_body, output_path, path_from_html):
                         img.attrib["style"] = " max-width:100%"
         return bool(imgs)
 
+    def calculate_root_from_asset(self, path_from_html, root_from_html):
+        """ get path to root from the downloaded/generated asset """
+
+        nb_jumps_root_from_html = root_from_html.count("../")
+        nb_back_jumps_output_path = path_from_html.count("../")
+        path_without_back_jumps = path_from_html[(nb_back_jumps_output_path) * 3 :]
```

Comment:
Where is this `3` coming from? Maybe add a comment.

Reply:
That's the length of one back jump, i.e. `len("../")` . Have used that instead, also added a comment.